### PR TITLE
Feat manjaro arm & deepin

### PR DESCRIPTION
**Pull request by bh90210**
#319
#320

### CODE REVIEW

File: cmd/cmd-mewn.go [hunk 1]
```diff
@@ -5,7 +5,7 @@ package cmd
...
...4a4badc051e0fa076b544951725f935b52e1d351473904da7d9852945fb1ac85bf898b7ebcfa3e07ff1871bc915006ed432abfd6deaffc36565aabe95382da517e8f41ca95121c38d694e1c5f6321cc4fd6263dc6b2a45521dad2c9a7fa552c0bbdf3e5dbf173a3c6fd37e31d67e22a90db5d6f824b6b5b3c8fdb8843a4cf069c5fe6dac18fc9a2ccd2c52ffec91ceaa4fc66356420fe2d9bbb2e85cc611ee5fe0ce2a16252ad49eda45529b7c59e42638f8f0f0dfa8578d6690c1d1cad39aee7e4e6067175921366265ec8b7616a17709eaec329f8ca892c8686591f88711d5f10df7b8c1f5b96c1ce90756ea300e187892597585ea4433ea0ac7be83e5ba13f4f5e56d7b1493a807eb77eb1b4c78d3195f838c69196a6f5ccae3d6ce92c93fa1f61f1ee0dcc9bc1ca11741f56ece18619797a0688d27bb7e0d099b9a470472a2f41ef161839274d1ebef58c3178eab81034b607cfe3dc4be559081f435d098fa112b6369aa84d6ce5209bf24a81d35b0f567eb5b1d6ce01b53fddf32fc265b8adf6c23a796cd5386472b4b7ebfb56c9ea1cfc32cee28ff27b8863245359087b872af4c8fd93991a25b9dbfc3270adda87d3e8c0d0e73db67b1982f538fd3d7b78764f8449a04b036a3cdf296bc92d6ad2b576e9b1c1f40b6a5f6f0beb1216a8aea2fd49aa4b27364d178df62e528e14c3a09b56aa4353a73ea256f7fbf3faf5a1ea7ed41505b3b8bc677096aaabca9ad3738a1392c4cb0e016cea3facec56aae29c7184acad11e39604c63d8abe31e02bdf3b6581cd65951a1e5d8addcb7b192431b6bc79256bf62fa370000ffff57890681cc1b0000")
```
Comment (bh90210) [inline cmd/cmd-mewn.go]: I honestly have no idea why is this happening.
I deleted everything, forked & cloned anew. 
when I run `mewn` within `wails/cmd` the generated file is this.

please advise.

Comment (leaanthony) [reply cmd/cmd-mewn.go]: No idea. Hack would be to copy the new yaml file line and Pate over old one. Let's just remove embedding this file altogether. I can do that if you like.

Comment (bh90210) [reply cmd/cmd-mewn.go]: I'll give it a try :+1:

**Comment by leaanthony** [comment]
This is awesome!

**Pull request merged**